docusaurus/docs/logo-resources.md

```diff
@@ -0,0 +1,30 @@
+---
+id: logo-resources
+title: Logo Resources
+sidebar_label: Logo Resources
+---
+
+These are the resources used to create the React logo.
```

Comment:
Minor; What do you think about emphasize Create React App logo rather than React logo

Reply:
That would be more accurate